From: Matthias Bach <marix@marix.org>
Date: 2015-01-02 19:23:21.825723016 +0100
Subject: Fix compilation of QR-code support
References: https://github.com/gridcoin/Gridcoin-Research/pull/13
Upstream: submitted

Support for QR codes could not be compiled as different names were used
for the feature toggle at different places.

diff --git a/gridcoinstake.pro b/gridcoinstake.pro
index 733fbd1..7f9a54e 100644
--- a/gridcoinstake.pro
+++ b/gridcoinstake.pro
@@ -65,7 +65,7 @@ win32:QMAKE_LFLAGS *= -Wl,--large-address-aware
 
 # use: qmake "USE_QRCODE=1"
 # libqrencode (http://fukuchi.org/works/qrencode/index.en.html) must be installed for support
-contains(USE_XQRCODE, 1) {
+contains(USE_QRCODE, 1) {
     message(Building with QRCode support)
     DEFINES += USE_QRCODE
     LIBS += -lqrencode
diff --git a/src/qt/qrcodedialog.cpp b/src/qt/qrcodedialog.cpp
index 219da39..2d1a6da 100644
--- a/src/qt/qrcodedialog.cpp
+++ b/src/qt/qrcodedialog.cpp
@@ -9,10 +9,7 @@
 #include <QPixmap>
 #include <QUrl>
 
-
-#if WINQR
 #include <qrencode.h>
-#endif
 
 QRCodeDialog::QRCodeDialog(const QString &addr, const QString &label, bool enableReq, QWidget *parent) :
     QDialog(parent),
@@ -58,7 +55,6 @@ void QRCodeDialog::genCode()
     if (uri != "")
     {
         ui->lblQRCode->setText("");
-#if WINQR
 
         QRcode *code = QRcode_encodeString(uri.toUtf8().constData(), 0, QR_ECLEVEL_L, QR_MODE_8, 1);
         if (!code)
@@ -82,7 +78,6 @@ void QRCodeDialog::genCode()
         ui->lblQRCode->setPixmap(QPixmap::fromImage(myImage).scaled(300, 300));
 
         ui->outUri->setPlainText(uri);
-#endif
     }
 }
 
@@ -90,7 +85,6 @@ QString QRCodeDialog::getURI()
 {
     QString ret = QString("gridcoin:%1").arg(address);
     int paramCount = 0;
-	#if WINQR
 
     ui->outUri->clear();
 
@@ -134,7 +128,6 @@ QString QRCodeDialog::getURI()
 
     ui->btnSaveAs->setEnabled(true);
     return ret;
-#endif
 }
 
 void QRCodeDialog::on_lnReqAmount_textChanged()


